tc_linux.go

```diff
@@ -31,7 +30,7 @@ const (
 
 // unlockpt unlocks the slave pseudoterminal device corresponding to the master pseudoterminal referred to by f.
 // unlockpt should be called before opening the slave side of a pty.
-func unlockpt(f *os.File) error {
+func unlockpt(f File) error {
```

Comment:
Wondering if we can rather drop the `File` interface and just use `*os.File`, assuming that it is the only implementation of the interface

Reply:
hmm it was introduced due to the GC stuff
- https://github.com/containerd/console/pull/33

Reply:
Yeah this is an remarkably common issue to run into with `*os.File`. It's a shame there isn't a properly supported way of doing this other than hacking around with `runtime.SetFinalizer` which is (imho) incredibly dodgy.

To be fair, even Rust only really solved this issue in 1.63 with `OwnedFd` and `BorrowedFd`...